src/openzaak/components/besluiten/openapi.yaml

```diff
@@ -1368,12 +1368,10 @@ paths:
           format: uri
       - name: informatieobject
         in: query
-        description: URL-referentie naar het INFORMATIEOBJECT (in de Documenten API)
```

Comment:
Please put this back

See: https://github.com/VNG-Realisatie/gemma-zaken/blob/master/api-specificatie/brc/1.0.x/openapi.yaml#L1395